src/Misc/layoutbin/runsvc.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [ -f ".Path" ]; then
+    # configure
+    export PATH=`cat .Path`
+    echo ".Path=${PATH}"
+fi
+
+# insert anything to setup env when running as a service
```

Comment:
Consider: Without an extra level of indirection to run the customization, this locks us in to the layout `+./externals/node/bin/node ./bin/AgentService.js`